docs/build-on-linea/gas-fees.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Gas fees on Linea
+sidebar_position: 6
+---
+
+ If you're familiar with gas fees on Ethereum, then you know that they heavily fluctuate depending on how busy the network is (for a refresher on gas click [here](https://support.metamask.io/hc/en-us/articles/4404600179227-User-Guide-Gas#:~:text=A%20normal%20transaction%20sending%20ETH,transactions%20also%20cost%2021%2C000%20gas.)). Linea's gas fees depend on Ethereum's fee, and we will explain how the L2 fees are calculated, but the **TLDR is that Linea's gas fees should be around 1/15th of Ethereum's, and we hope to reduce them even further in the future.**
+
+## How do I check the gas price on Linea?
+
+ If you want to check the gas price on Linea, run the code below.
+
+``` bash
+curl https://linea-mainnet.infura.io/v3/your-infura-api-key \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -d '{"jsonrpc":"2.0","method":"eth_gasPrice","params": [],"id":1}'
+```
+:::note
+The output of the request returns a hexadecimal equivalent of an integer representing the current gas price in wei. Convert the hexadecimal value into decimals to get the wei value. You can use any hexadecimal to decimal converter such as [RapidTables](https://www.rapidtables.com/convert/number/hex-to-decimal.html).
+
+:::
+
+For more information on this method, take a look at [eth_gasPrice JSON_RPC](https://docs.infura.io/networks/ethereum/json-rpc-methods/eth_gasprice).
+
+You can also use the eth_gasprice method on Ethereum and then compare the two values to see if L2 gas price = ~L1/15 gas price.
+
+``` bash
+curl https://mainnet.infura.io/v3/your-infura-api-key \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -d '{"jsonrpc":"2.0","method":"eth_gasPrice","params": [],"id":1}'
+```
+
+## The Formula
+
+**Total L2 Fee Equation**
+
+$$
+\text{total\_L2\_fee} = \text{L2\_base\_fee} + \text{L2\_miner\_tip}
+$$
+
+The L2 fee equation is comprised of two parts, the base fee and the miner tip. The base fee is 7 wei, which is the minimum base fee a standard protocol allows. The miner tip equation is a bit complex and will be explained in more detail below.
+
+
+**L2 Miner Tip Equation**
+
+$$
+l2\_miner\_tip = \left( \sum_{i} (baseFee[i] \times 0.066 + reward[i] \times 0.066) \times ratio[i] \right) \div \left( \sum_{i} ratio[i] \right)
+$$
```

Comment:
Can you please share a screenshot of how this formula is renderend? Not familiar with this notation so won't be able to tell.

Reply:
https://doc-zk-evm-git-gas-fees-infura-web.vercel.app/build-on-linea/gas-fees you can look at the preview of the page to see what it looks like!